<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuronizeClass</class>
 <widget class="QMainWindow" name="NeuronizeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neuronize</string>
  </property>
  <property name="windowIcon">
   <iconset resource="neuronize.qrc">
    <normaloff>:/Neuronize/icons/neuronize.png</normaloff>:/Neuronize/icons/neuronize.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget_MainContainer">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_SomaCreator">
       <attribute name="title">
        <string>Generate Neuron</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_SomaCreator"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_RepairMeshes">
       <attribute name="title">
        <string>Repair Meshes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_RepairMeshes"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_CompareMeshes">
       <attribute name="title">
        <string>Compare Meshes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_CompareMeshes"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_SomaGenerator">
       <attribute name="title">
        <string>Soma builder</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_SomaDeformer"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_DendritesGenerator">
       <attribute name="title">
        <string>Dendrites/Spines builder</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_DendritesGenerator"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_SpinesGenerator">
       <attribute name="title">
        <string>Spines builder</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_SpinesGenerator"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGenerateNewNeuron"/>
    <addaction name="actionRepair_Meshes"/>
    <addaction name="actionCompare_Meshes"/>
    <addaction name="actionExport_Neuron_info"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionTake_a_snapshot"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout_Neuronize"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionReset_Python_Enviorement"/>
    <addaction name="actionExport_Database_File"/>
    <addaction name="actionDelete_Local_Database"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGenerateNewNeuron"/>
   <addaction name="actionRepair_Meshes"/>
   <addaction name="actionCompare_Meshes"/>
   <addaction name="actionUndo"/>
   <addaction name="actionTake_a_snapshot"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionGenerateNewNeuron">
   <property name="icon">
    <iconset resource="neuronize.qrc">
     <normaloff>:/Neuronize/icons/document-open-6.png</normaloff>:/Neuronize/icons/document-open-6.png</iconset>
   </property>
   <property name="text">
    <string>Generate new Neuron</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="neuronize.qrc">
     <normaloff>:/Neuronize/icons/application-exit.png</normaloff>:/Neuronize/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="neuronize.qrc">
     <normaloff>:/Neuronize/icons/edit-undo-8.png</normaloff>:/Neuronize/icons/edit-undo-8.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionTake_a_snapshot">
   <property name="icon">
    <iconset resource="neuronize.qrc">
     <normaloff>:/Neuronize/icons/Camera.app.png</normaloff>:/Neuronize/icons/Camera.app.png</iconset>
   </property>
   <property name="text">
    <string>Take a snapshot</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="neuronize.qrc">
     <normaloff>:/Neuronize/icons/help-2.png</normaloff>:/Neuronize/icons/help-2.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout_Neuronize">
   <property name="text">
    <string>About Neuronize</string>
   </property>
  </action>
  <action name="actionBatchBuilder">
   <property name="text">
    <string>BatchBuilder</string>
   </property>
  </action>
  <action name="actionRepair_Meshes">
   <property name="icon">
    <iconset resource="neuronize.qrc">
     <normaloff>:/Neuronize/icons/repair.png</normaloff>:/Neuronize/icons/repair.png</iconset>
   </property>
   <property name="text">
    <string>Repair Meshes</string>
   </property>
  </action>
  <action name="actionCompare_Meshes">
   <property name="icon">
    <iconset resource="neuronize.qrc">
     <normaloff>:/Neuronize/icons/compare.png</normaloff>:/Neuronize/icons/compare.png</iconset>
   </property>
   <property name="text">
    <string>Compare Meshes</string>
   </property>
  </action>
  <action name="actionExport_Neuron_info">
   <property name="text">
    <string>Export Neuron Info</string>
   </property>
  </action>
  <action name="actionReset_Python_Enviorement">
   <property name="text">
    <string>Reset Python Enviorement</string>
   </property>
  </action>
  <action name="actionDelete_Local_Database">
   <property name="text">
    <string>Delete Local Database</string>
   </property>
  </action>
  <action name="actionExport_Database_File">
   <property name="text">
    <string>Export Database File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="neuronize.qrc"/>
 </resources>
 <connections/>
</ui>
